spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -133,6 +223,56 @@
         expect(pohandler).to receive(:update_version_after_validation)
         c2m.check_catalog_version
       end
+
+      context 'check whether PreservedCopy already has a status other than OK_STATUS, re-check status if so' do
+        it 'had OK_STATUS, version increased, should still have OK_STATUS' do
+          pres_copy.status = PreservedCopy::OK_STATUS
+          pres_copy.save!
+          allow(c2m).to receive(:moab_validation_errors).and_return([])
+          c2m.check_catalog_version
+          expect(pres_copy.reload.status).to eq PreservedCopy::OK_STATUS
+        end
+        it 'had OK_STATUS, but is now INVALID_MOAB_STATUS' do
+          skip "validate if version increases on disk? as in POH#check_existence, checks status iff pres copy not OK."
```

Comment:
in `PreservedObjectHandler#check_existence`, we validate explicitly when the incoming version is greater than the catalog version.  that was already happening in that branch, and so `set_status_as_seen_on_disk` was only called if the object already had a non-OK status.  i'm inclined to say we should just call `set_status_as_seen_on_disk` regardless of the current catalog status when incoming version is greater than catalog version.  this seems simple and reasonable, and like it accomplishes similar behavior to the M2C counterpart with less code than what's in this PR now.

Reply:
I'm okay with your suggestion;  it's really up to @julianmorley .   I'd suggest coding what you think is sensible here, following up with Julian, and make a separate ticket if he wants something different?  (Or just get an answer from him RIGHT NOW)